avocado/utils/memory.py

```diff
@@ -41,12 +41,14 @@ def _check_memory_state(block):
     Check the given memory block is online or offline
 
     :param block: memory block id.
-    :type string: like 198
+    :type string: like 198 or memory198
     :return: 'True' if online or 'False' if offline
     :rtype: bool
     """
+    if not block.startswith("memory"):
```

Comment:
This is what's causing the Travis failure. You have to add the extra space in this commit (as that is where you add the code which needs to be indented).

Reply:
Actually it'd look better to keep the `def _is_online` where it was and add the `if not block.startswith("memmory")` after the `def _is_online`